Modules/Libs/QuestieLib.lua

```diff
@@ -548,3 +548,18 @@ function QuestieLib:UnpackBinary(val)
     end
     return ret
 end
+
+function QuestieLib:ArrayToDict(arr)
+    local t = {}
+    if arr == nil then
+        Questie:Debug(DEBUG_DEVELOP, "ArrayToDict parameter is nil")
+    elseif #arr % 2 ~= 0 then
+        Questie:Debug(DEBUG_DEVELOP, "ArrayToDict parameter is empty")
```

Comment:
Since these paths only lead to log messages: Do we need them? Especially in a lib method, that could clutter the log quite easily 🤔

Reply:
I added them while debugging the method itself, but you're right, they're not strictly necessary. I could remove them, or we could put them at the SPAM level. I'm open to either. Do you have a preference?

Reply:
I ended up putting them at spam and updated the method a bit:
https://github.com/Questie/Questie/pull/2566/commits/6e68f63e3db271dea5e4931db556b4eb830852ed